tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -872,16 +872,24 @@ hash/test_generic_hash.py::test_backend_error_messages:
 
 hash/test_generic_hash.py::test_ecmp_and_lag_hash:
   skip:
-    reason: 'On Mellanox SPC1 platforms, due to HW limitation, it would not support CRC_CCITT algorithm'
+    reason: 'On Mellanox SPC1 platforms, due to HW limitation, it would not support CRC_CCITT algorithm. For broadcom, ECMP/LAG hash not supported in broadcom SAI'
+    conditions_logical_operator: or
     conditions:
       - "asic_gen == 'spc1'"
+      - "asic_type in ['broadcom']"
 
 hash/test_generic_hash.py::test_ecmp_and_lag_hash[CRC-INNER_IP_PROTOCOL:
   skip:
     reason: "On Mellanox platforms, due to HW limitation, it would not support CRC algorithm on INNER_IP_PROTOCOL field"
     conditions:
     - "asic_type in ['mellanox']"
 
+hash/test_generic_hash.py::test_ecmp_hash:
+  skip:
+    reason: 'ECMP hash not supported in broadcom SAI'
+    conditions:
+      - "asic_type in ['broadcom']"
```

Comment:
The purpose of this condition is to skip all test cases that begin with `hash/test_generic_hash.py::test_ecmp_hash` on broadcom. Due to the principle of "longest matching" in conditional markers, we only apply conditions from the longest matching prefix when multiple prefixes share the same marker, such as `skip`.

In this situation, since there are longer prefixes, such as `hash/test_generic_hash.py::test_ecmp_hash[CRC-INNER_IP_PROTOCOL`, we should also incorporate this condition into those longer prefixes to take effect.

Reply:
Fixed, thanks.